src/common/base.scss

```diff
@@ -363,8 +363,6 @@ button {
   padding: 4px 20px;
   text-align: center;
   font-size: 16px;
-  word-break: break-all;
-  word-wrap: break-word;
```

Comment:
It is a bad idea to have these styles global, but removing them may break something. I have checked all screens briefly, they look ok.